src/frameworks/claude/claude-converter.ts

```
@@ -0,0 +1,109 @@
+  ): Promise<EvaluationRequest> {
+    let {
+      messages: requestMessages,
+      available_tools: requestAvailableTools,
+    } = await this.convertRequest(request);
+
+    const messages: LLMMessage[] = requestMessages || [];
+    const available_tools: LLMToolDefinition[] = requestAvailableTools || [];
+
+    // Check if response is streaming or non-streaming
+    if (this.isClaudeStreamingChunk(response)) {
+      let streamingResultMessages = await this.handleStreaming(response);
```

ESLint

[error] 'streamingResultMessages' is never reassigned. Use 'const' instead.

(prefer-const)